### ISSUE TITLE
Add Stimulus.plot

### ISSUE THREAD

**Issue by mbeyeler** [pulse2percept/pulse2percept#173]
Write a `plot` method for the `Stimulus` object:
```
def plot(self, electrodes=None, time=None):
    """Plot the stimulus

    Parameters
    ------------
    electrodes : int, string, or list thereof; optional, default: None
        The electrodes for which to plot the stimulus. If None, all electrodes are plotted.
    time : float, or list thereof; optional, default: None
        The time points at which to plot the stimulus. If None, all time points are plotted.

    Returns
    --------
    ax : matplotlib.axes.Axes or np.ndarray of them
        Returns one matplotlib.axes.Axes per electrode
    """
```

Some specifics:
* `stim.plot()` without arguments should plot all the data
* For each electrode, make a line plot of the data in a subplot (check out the [gallery](https://pulse2percept.readthedocs.io/en/latest/examples/stimuli/plot_pulses.html) for plotting examples, but put different electrodes on rows not columns)
* Each electrode should be a row in the plot
* Make sure to label the axes (time in seconds and current in uA)
* You can use the `__getitem__` functionality to automatically do slicing and interpolating of time

**Comment by arokem** [comment]
Potentially check out the plotting in MNE-Python for inspiration. For example here: https://mne.tools/dev/auto_examples/io/plot_objects_from_arrays.html

**Comment by mbeyeler** [comment]
Closed via #176
